### PR TITLE
Various vanilla sound mapping fixes

### PR DESCRIPTION
**Pull request by letsgoawaydev**
This PR fixes:
- UI Toast playsounds
- Various copper item playsounds
- Various coral playsounds
- Cobweb playsounds
- other misc playsounds that i found

### CODE REVIEW

File: sounds.json [hunk 1]
```diff
@@ -670,7 +670,9 @@
     "playsound_mapping": "mob.bogged.hurt",
     "bedrock_mapping": ""
   },
-  "entity.bogged.shear": {},
+  "entity.bogged.shear": {
+    "playsound_mapping": "mob.sheep.shear"
+  },
```
Comment (onebeastchris) [inline sounds.json:675]: are the sounds for sheep/bogged the same?

File: sounds.json [hunk 2]
```diff
@@ -1353,29 +1356,38 @@
...
+  "block.copper_grate.hit": {
+    "playsound_mapping": "step.copper_grate",
+    "bedrock_mapping": "HIT"
+  },
+  "block.copper_grate.fall": {
+    "playsound_mapping": "step.copper_grate"
+  },
+  "block.copper_trapdoor.close": {
+    "playsound_mapping": "open_trapdoor.copper"
+  },
```
Comment (onebeastchris) [inline sounds.json:1368]: Would "close_door.copper" be closer to the copper trapdoor closing sound? Or are they the same on java?

File: sounds.json [hunk 3]
```diff
@@ -1353,29 +1356,38 @@
...
     "bedrock_mapping": "FALL"
   },
   "block.coral_block.hit": {
-    "playsound_mapping": "hit.stone",
+    "playsound_mapping": "use.coral",
     "bedrock_mapping": "HIT"
   },
   "block.coral_block.place": {
-    "playsound_mapping": "use.stone",
+    "playsound_mapping": "dig.coral",
```
Comment (onebeastchris) [inline sounds.json:1385]: What's the reasoning for not using use.coral here? Just curious; haven't tested it myself

**Review by onebeastchris — APPROVED** [review]

File: sounds.json (hunk 1 above)
Comment (letsgoawaydev) [reply sounds.json:675]: Yes, here is the sound from the Java 1.21 sounds.json.
```json
 "entity.bogged.shear": {
    "sounds": [
      "mob/sheep/shear"
    ],
    "subtitle": "subtitles.item.shears.shear"
  },
  ```

File: sounds.json (hunk 2 above)
Comment (letsgoawaydev) [reply sounds.json:1368]: The closing and opening sounds for the trapdoor are the exact same on java and bedrock. However, `close_trapdoor.copper` does not exist, so I have to use this id.

File: sounds.json (hunk 3 above)
Comment (letsgoawaydev) [reply sounds.json:1385]: On Java, the dig sound is used.
```json
  "block.coral_block.place": {
    "sounds": [
      "dig/coral1",
      "dig/coral2",
      "dig/coral3",
      "dig/coral4"
    ],
    "subtitle": "subtitles.block.generic.place"
  },
  ```

**Pull request merged**